obspy/io/seiscomp/core.py

```diff
@@ -87,6 +87,9 @@ def validate(path_or_object, version='0.9', verbose=False):
     :rtype: bool
     :return: `True` if SC3ML file is valid.
     """
+    if version is None:
+        version = SUPPORTED_XSD_VERSION[-1]
```

Comment:
Could we not also look up the version from the file that is to be validated?

Reply:
You're right! I changed that.